.github/workflows/ci.yml

```diff
@@ -15,9 +15,6 @@ jobs:
           - machine: windows-2022
             containers: windows
             log-dir: "/c/ProgramData/OpenTelemetry .NET AutoInstrumentation/logs"
-          - machine: ubuntu-20.04
```

Comment:
I think that it is worth to keep it, at least for now.
It is the only CI build pipeline executing IntegrationTests with Linux Docker container.

Reply:
reverted in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2203/commits/f3bd5c429c7f42ab801b38211f8348d27c064e02